scripts/zones/Ceizak_Battlegrounds/mobs/Unfettered_Twitherym.lua

```diff
@@ -0,0 +1,119 @@
+-----------------------------------
+-- Area: Ceizak Battlegrounds
+-- NM: Unfettered Twitherym
+-- !pos 210 0 115 261
+-- !additem 6011
+-----------------------------------
+---@type TMobEntity
+local entity = {}
+
+local originalDamage = 0
+local damageReductionValue = -9000
+
+local skillchainAffinities = {
+    Fire = { xi.skillchainType.FUSION, xi.skillchainType.LIQUEFACTION, xi.skillchainType.LIGHT, xi.skillchainType.LIGHT_II },
+    Water = { xi.skillchainType.REVERBERATION, xi.skillchainType.DISTORTION, xi.skillchainType.DARKNESS, xi.skillchainType.DARKNESS_II },
+    Thunder = { xi.skillchainType.IMPACTION, xi.skillchainType.FRAGMENTATION, xi.skillchainType.LIGHT, xi.skillchainType.LIGHT_II },
+    Wind = { xi.skillchainType.DETONATION, xi.skillchainType.FRAGMENTATION, xi.skillchainType.LIGHT, xi.skillchainType.LIGHT_II },
+    Ice = { xi.skillchainType.INDURATION, xi.skillchainType.DISTORTION, xi.skillchainType.DARKNESS, xi.skillchainType.DARKNESS_II },
+    Earth = { xi.skillchainType.SCISSION, xi.skillchainType.GRAVITATION, xi.skillchainType.DARKNESS, xi.skillchainType.DARKNESS_II },
+    Dark = { xi.skillchainType.COMPRESSION, xi.skillchainType.DARKNESS, xi.skillchainType.DARKNESS_II },
+    Light = { xi.skillchainType.TRANSFIXION, xi.skillchainType.LIGHT, xi.skillchainType.LIGHT_II }
+}
+
+local chosenElement = nil
```

Comment:
If you used a local var on the mob, you wouldnt need to define a local variable outside of the mobs functions.